src/main/kotlin/com/openlattice/graph/GraphApi.kt

```diff
@@ -33,13 +33,9 @@ const val CONTROLLER = "/graph"
 const val BASE = SERVICE + CONTROLLER
 //@formatter:on
 
-const val NEIGHBORS = "neighbors"
+const val NEIGHBORS = "neighbors" // TODO this needs a slash
```

Comment:
seems like a simple TODO to complete?

Reply:
:D I know, but since right now it does not have, I'm not sure if any APPs are using it like this or not and could break things if they are using it.

Reply:
ah okay gotcha